src/main/java/com/stuypulse/robot/constants/ShotMap.java

```diff
@@ -14,15 +14,21 @@
 /** Contains interpolation tables for shooting and alignment */
 public interface ShotMap {
 
```

Comment:
add an interface here called Distances that goes smth like this 
{
   double  RING = RING_DISTANCE.get() 
  double INTERP_A = Units.inchesToMeters(167)
    double INTERP_B = Units.inchesToMeters(184)
  ...
    double PAD = PAD_DISTANCE.get() 
 }
 
 and use the constants in the two interfaces (also maybe think fo something better than INTERP_A)